### PR TITLE
[v6r15] Store in DB on setup

### PR DESCRIPTION
**Pull request by Sbalbp**
Modified ComponentInstaller to add entries to the InstalledComponents DB for services, agents and executors installed during the setup process.

### CODE REVIEW

File: FrameworkSystem/Client/ComponentInstaller.py [hunk 1]
```diff
@@ -1678,18 +1678,27 @@ def setupSite( self, scriptCfg, cfg = None ):
       result = self.setupComponent( 'service', system, service, extensions, monitorFlag = False )
       if not result['OK']:
         gLogger.error( result['Message'] )
+      result = MonitoringUtilities.monitorInstallation( 'service', system, service )
+      if not result['OK']:
+        self.__errMsg( 'Error registering installation into database: %s' % result[ 'Message' ] )
```
Comment (fstagni) [inline FrameworkSystem/Client/ComponentInstaller.py]: __errMsg is not a function in this module.

File: FrameworkSystem/Client/ComponentInstaller.py [hunk 2]
```diff
@@ -1679,18 +1679,27 @@ def setupSite( self, scriptCfg, cfg = None ):
       result = self.setupComponent( 'service', system, service, extensions, monitorFlag = False )
       if not result['OK']:
         gLogger.error( result['Message'] )
+      result = MonitoringUtilities.monitorInstallation( 'service', system, service )
```
Comment (atsareg) [inline FrameworkSystem/Client/ComponentInstaller.py]: Should it be called even if setupComponent() call fails ? Same for other component types

Comment (Sbalbp) [inline FrameworkSystem/Client/ComponentInstaller.py]: No, it shouldnt. Thanks for pointing out.

**Comment by atsareg** [comment]
Review OK

**Pull request merged**